examples/integrations/gcp/src/coordinator/training/images/bounding_box_training_placeholder.py

```diff
@@ -14,8 +17,28 @@ def parse_args(self, json_data: Dict[str, Any]) -> str:
 
     def run_local(self, json_data: Dict[str, Any]) -> JobStatus:
         training_file_uri = self.parse_args(json_data)
-        sleep(1)
-        return JobStatus(JobState.SUCCESS)
+        dataset_display_name = f"bounding_box_{uuid4()}"
+        dataset = aiplatform.ImageDataset.create(
+            display_name=dataset_display_name,
+            gcs_source=[training_file_uri],
+            import_schema_uri=aiplatform.schema.dataset.ioformat.image.
+            bounding_box)
+        job = aiplatform.AutoMLImageTrainingJob(
+            display_name="matt-test-train-automl",
+            prediction_type="object_detection",
+            model_type="MOBILE_TF_LOW_LATENCY_1")
+        model = job.run(
+            dataset=dataset,
+            # Every 1,000 is 1 node-hour. This is close to the minimum (20k).
+            # Increase / parameterize this before deploying
+            budget_milli_node_hours=1000,
```

Comment:
the interface doesn't let you budget less than 8 node hours, is this possible via the API?

Reply:
This is the min before the api complains (I set this at 500 and it threw an excpetion). Although it ran longer than the budget allowed. So I am not really sure how this is enforced.